src/components/ConfigureEmail.tsx

```diff
@@ -79,26 +71,27 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
   const deleteEmail = async (event: React.SyntheticEvent, email: string) => {
     event.preventDefault()
     try {
-      let deletedEmailList = emails?.filter(item => item !== email)
-      setEmails(deletedEmailList)
```

Comment:
Was previously updating the client UI, providing some potential for misrepresenting what was on the server (and it would if there were any errors in the call!).  Now we refetch the data after update